spec/latest/json-ld-api/index.html

```diff
@@ -4633,7 +4633,15 @@ <h3>RemoteDocument</h3>
           <a data-link-for="JsonLdErrorCode">multiple context link headers</a>.</dd>
         <dt><dfn data-dfn-for="RemoteDocument">documentUrl</dfn></dt>
         <dd>The final URL of the loaded document. This is important
-          to handle HTTP redirects properly.</dd>
+          to handle HTTP redirects properly.
+          <span class="changed">Loading a document with a redirect MUST use the redirected URL as the <a>documentUrl</a>.</span>
+          <div class="note">If the <a>JSON-LD document</a> is retrived via <em>XMLHttpRequest</em>
+            and the server issues an HTTP redirect, the document location
+            MUST reflect the redirected location rather than that originally
+            requested. The redirected location may be provided via the <a
+            data-cite="XHR#dom-xmlhttprequest-responseurl">responseURL</a>
```

Comment:
I think most browsers already adopted Fetch API, maybe also worth mentioning it here?
https://fetch.spec.whatwg.org/#dom-response-url

Reply:
@elf-pavlik care to suggest some text?

Reply:
I'll double check how https://fetch.spec.whatwg.org and https://xhr.spec.whatwg.org/ stand at current time and possibly will follow up with separate PR to reference Fetch.

Reply:
I'm getting concerned with the amount of specifics, even though in notes, with other specs. Perhaps this would be better to be more vague about using facilities such as XHR and Fetch to retrieve the redirected URL, rather than discuss specific mechanisms.